interop-testing/src/main/java/io/grpc/testing/integration/Http2Client.java

```diff
@@ -272,6 +272,7 @@ private void rstDuringData() throws Exception {
 
     private void goAway() throws Exception {
       assertResponseEquals(blockingStub.unaryCall(simpleRequest), goldenResponse);
+      TimeUnit.SECONDS.sleep(1);
```

Comment:
At the moment, there is no verification on the client-side. The server asserts that the second call is seen on another stream: https://github.com/grpc/grpc/issues/9300 and https://github.com/grpc/grpc/blob/master/doc/negative-http2-interop-test-descriptions.md